app/src/main/java/sistemas/puc/com/finantialapp/FetchIndicesTask.java

```diff
@@ -0,0 +1,212 @@
+package sistemas.puc.com.finantialapp;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.net.Uri;
+import android.os.AsyncTask;
+import android.util.Log;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+import sistemas.puc.com.finantialapp.data.FinantialContract;
+import sistemas.puc.com.finantialapp.model.IndiceEnum;
+import sistemas.puc.com.finantialapp.util.Util;
+
+public class FetchIndicesTask extends AsyncTask<Void, Void, List<ContentValues>>{
+
+    private final static String BCB_VALUE = "valor";
+    private final static String BCB_DATE = "data";
+
+    private final static String IBGE_VALUE = "v";
+    private final static String IBGE_DATE = "p_cod";
+    private final static int IBGE_MENSAL = 0;
+    private final static int IBGE_ANUAL = 1;
+
+    private static final String SELIC_URL = "http://api.bcb.gov.br/dados/serie/bcdata.sgs.1178/dados/ultimos/1?formato=json";
+    private static final String IPCA_URL = "http://servicodados.ibge.gov.br/api/v2/conjunturais/1419/periodos/-1/indicadores";
+    private static final String INPC_URL = "http://servicodados.ibge.gov.br/api/v2/conjunturais/1100/periodos/-1/indicadores";
+
+    private final String LOG_TAG = FetchIndicesTask.class.getSimpleName();
+    private Context m_context;
+
+    public FetchIndicesTask(Context context) {
+        m_context = context;
+    }
+
+    @Override
+    protected void onPreExecute(){}
+
+    @Override
+    protected List<ContentValues> doInBackground(Void... voids) {
+
+        String selicJsonStr = getJsonStr(SELIC_URL);
+        String inpcJsonStr = getJsonStr(INPC_URL);
+        String ipcaJsonStr = getJsonStr(IPCA_URL);
+
+        //placeholders
+
+        ContentValues CDI = new ContentValues();
+        setupIndiceCV(CDI, IndiceEnum.CDI);
```

Comment:
Make the function return the ContentValues like that:
`ContentValues CDI = setupIndiceCV(CDI, IndiceEnum.CDI);`